Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 The Service Manager Authors
```

Comment:
not a part of this PR IMO. This is a general task for all repositories that have a makefile and should be addressed separately

Reply:
sure, should we create an issue or several and it which repo ?

Reply:
Created - https://github.com/Peripli/service-manager/issues/170 
It would be a big task if we decide to add provisioning of dev environments (cf dev start, minikube start, etc.)